src/typechecker.erl

```diff
@@ -4042,6 +4136,10 @@ print_error(Error, Opts) ->
 handle_type_error({type_error, Expression, ActualType, ExpectedType}, Opts)
   when is_tuple(Expression) ->
     print_type_error(Expression, ActualType, ExpectedType, Opts);
+handle_type_error({nonexhaustive, Anno, Example}, Opts) ->
+    io:format("Nonexhaustive patterns starting on line ~p.~n"
+	      "Example values which is not covered: ~p~n"
+	     ,[format_location(Anno, brief, Opts), Example]);
```

Comment:
Note that `format_location(Anno, brief, Opts)` returns the empty string if some option is set to show non-brief location.

Reply:
![image](https://user-images.githubusercontent.com/152020/64995259-a6855b00-d8d2-11e9-8bb6-7488100fa769.png)